### DISCUSSION TITLE
Reset Password - Change or remove temp password expiration

### DISCUSSION THREAD

**Discussion by kellis5137** [Budibase/budibase#6187] [Help]
Hi,
I'm wondering if there is anyone to configure budibase to have a longer expiration date when resetting passwords or to remove the expiration completely?

Keith

**Comment by melohagan** [comment]
Hi @kellis5137 

When an admin resets a user account password, there is no expiration. They just need to enter a new password whenever they next login.

The timeout only applies when resetting your own account password. 

I'm wondering why you want to remove the expiration - what problem are you trying to solve?

**Reply by kellis5137** [reply]
Hmm, interesting. So I created users via the Restful API and some people were saying that their temp passwords were showing up as expired. I then ran a script to use the Restful API and similar. I figured it must be that they were expiring, similar to the "reset password" expiration. However, if you're telling me it's not, there must be something else going on. Thank you!
